### ISSUE TITLE
SVG Support for BaguetteBox

### ISSUE THREAD

**Issue by jackiboy** [feimosi/baguetteBox.js#97]
Hi I am currently using Baguette on my site for an Image gallery (Which is working great).

I have some SVG diagrams that I would like to Lightbox. I notice that BaguetteBox does not support SVG?

```
This Works 
<a href="img/image.jpg" data-caption="Golden Gate Bridge">
    <img src="img/thumbs/thumbnail.jpg">
</a>

This Does Not Work 
<a href="img/diagram.svg" data-caption="Golden Gate Bridge">
    <img src="img/thumbs/thumbnail.jpg">
</a>
```

Cheers,
Jack

**Comment by feimosi** [comment]
Hi @jackiboy !
You're right, it doesn't work by default. You have to adjust it with [a special _option_](https://github.com/feimosi/baguetteBox.js#customization).
Check out the `filter` and add `svg` to the regex pattern. It should work then.

**Comment by XhmikosR** [comment]
@feimosi: maybe we should add svg to the defaults.

**Comment by feimosi** [comment]
Yes, probably it should be by default.

**Comment by feimosi** [comment]
Closed in favor of #167.
